src/components/typography/typography.stories.tsx

```diff
@@ -133,6 +132,11 @@ export const TruncateStory: Story = () => (
         truncate.
       </Typography>
     </Box>
+    <Box height={80} width={80} backgroundColor="lightblue">
```

Comment:
**suggestion**(non-blocking): it could be worth seeing if the width of these box's could be extended to kind of give a better idea of the content, even though I know its meant to be truncated haha